modules/campaignion_flexible_form/campaignion_flexible_form_templates/campaignion_flexible_form_templates.features.uuid_node.inc

```diff
@@ -269,8 +269,12 @@ function campaignion_flexible_form_templates_uuid_features_default_content() {
     ),
     'conditionals' => array(),
   ),
-  'date' => '2012-07-23 16:10:30 +0200',
-  'user_uuid' => '59e16a84-5bfa-438b-be8f-782a0af74750',
+  'path' => array(
+    'pathauto' => FALSE,
+    'alias' => 'content/default-flexible-form-template-do-not-delete',
+  ),
```

Comment:
The pathauto settings seem a bit inconsistent to me, shouldn't it be either TRUE or FALSE for *all* content types? 
Also is the module always enabled?

Reply:
I’ve made it FALSE in all nodes.